src/box/lua/schema.lua

```diff
@@ -350,6 +352,12 @@ end
 
 box.is_in_txn = builtin.box_txn
 
+box.internal.memtx_tx_gc = function(step_num)
```

Comment:
Why not exporting memtx_tx_story_gc() instead?

Reply:
`memtx_tx_story_gc()` runs only `txm.must_do_gc_steps` number of steps, which is set to 0 after each the function's call.